jmi_erpnext/hooks.py

```diff
@@ -149,7 +149,7 @@
 			"Sales Invoice-jmi_customer_id"]],
 		]
 	},
-	{"dt": "Print Format", "filters": [["name", "in", ["Stationary - POS Invoice","JMI Offline Point of Sale","Mobile - POS Invoice"]]]},
+	{"dt": "Print Format", "filters": [["name", "in", ["Stationary - POS Invoice","JMI Offline Point of Sale","Mobile - POS Invoice","JMI Online Point of Sale","POS Invoice"]]]},
```

Comment:
@Siddhisawant8 POS Invoice is a default print format, it's not needed in hooks. Also, JMI Online POS print format isn't needed here. Stationary POS and Mobile-POS is what we need here.